charts/pega/README.md

```diff
@@ -25,14 +25,15 @@ provider: "k8s"
 
 Use the `action` section in the helm chart to specify a deployment action.  The standard actions is to deploy against an already installed database, but you can also install a Pega system. These values are case-sensitive and must be lowercase.
 
-For additional, required installation parameters, see the [Installer section](#install).
+For additional, required installation parameters, see the [Installer section](#installations).
 
 Value             | Action
 ---               | ---
 deploy            | Start the Pega containers using an existing Pega database installation.
 install           | Install Pega Platform into your database without deploying.
 install-deploy    | Install Pega Platform into your database and then deploy.
-
+upgrade           | Upgrade or patch Pega Platform in your database without deploying.
+upgrade-deploy    | Upgrade or patch Pega Platform in your database without deploying.
```

Comment:
upgrade-deploy does upgrade process and then deploy.

Reply:
done